test/integration/voting.test.js

```diff
@@ -0,0 +1,217 @@
+    rank: 1,
+    rate: 1,
+    username: 'thepool',
+    vote: '3883699551759500',
+  },
+  {
+    address: '14593474056247442712L',
+    approval: 32.08,
+    missedblocks: 283,
+    producedblocks: 38035,
+    productivity: 99.26,
+    publicKey: 'ec111c8ad482445cfe83d811a7edd1f1d2765079c99d7d958cca1354740b7614',
+    rank: 2,
+    rate: 2,
+    username: 'liskpool_com_01',
+    vote: '3796476912180144',
+  },
+  {
+    address: '13943256167405531820L',
+    approval: 32.03,
+    missedblocks: 164,
+    producedblocks: 38088,
+    productivity: 99.57,
+    publicKey: '00de7d28ec3f55f42329667f08352d0d90faa3d2d4e62c883d86d1d7b083dd7c',
+    rank: 3,
+    rate: 3,
+    username: 'iii.element.iii',
+    vote: '3789594637157356',
+  },
+];
+
+const account = {
+  ...accounts.genesis,
+  delegate: {},
+  multisignatures: [],
+  u_multisignatures: [],
+  unconfirmedBalance: '0',
+};
+
+let store;
+let wrapper;
+let listDelegatesApiStub;
+let listAccountDelegatesStub;
+let voteApiStub;
+let accountAPIStub;
+let localStorageStub;
+
+const loginProcess = (votes = []) => {
+  accountAPIStub = stub(accountAPI, 'getAccount');
+  localStorageStub = stub(localStorage, 'getItem');
+  localStorageStub.withArgs('accounts').returns(JSON.stringify([{}, {}]));
+  listDelegatesApiStub = stub(delegateApi, 'listDelegates');
+  listAccountDelegatesStub = stub(delegateApi, 'listAccountDelegates');
+  voteApiStub = stub(delegateApi, 'vote');
+  store = prepareStore({
+    account: accountReducer,
+    peers: peersReducer,
+    voting: votingReducer,
+    transactions: transactionReducer,
+  }, [
+    thunk,
+    accountMiddleware,
+    loginMiddleware,
+    transactionsMiddleware,
+    peerMiddleware,
+  ]);
+
+  accountAPIStub.withArgs(match.any).returnsPromise().resolves({ ...account });
+  store.dispatch(activePeerSet({ network: getNetwork(networks.mainnet.code) }));
+  accountAPIStub.withArgs(match.any).returnsPromise().resolves({ ...account });
+  store.dispatch(accountLoggedIn(account));
+
+  listDelegatesApiStub.returnsPromise()
+    .resolves({ delegates, success: true, totalCount: 20 });
+  listAccountDelegatesStub.returnsPromise()
+    .resolves({ delegates: votes, success: true });
+
+  wrapper = mount(renderWithRouter(Voting, store));
+  expect(store.getState().account).to.be.an('Object');
+  expect(store.getState().voting).to.be.an('Object');
+  expect(store.getState().peers).to.be.an('Object');
+};
+
+const restoreApiMocks = () => {
+  listDelegatesApiStub.restore();
+  listAccountDelegatesStub.restore();
+  voteApiStub.restore();
+  accountAPIStub.restore();
+  localStorageStub.restore();
+};
+
+describe('@integration test of Voting', () => {
+  describe('Scenario: should allow to select delegates in the "Voting" and vote for them', () => {
+    step('I\'m logged in as "genesis"', () => { loginProcess(); });
+
+    step('And next button should be disabled', () => {
+      expect(wrapper.find('button.next').props().disabled).to.be.equal(true);
```

Comment:
This would be better as
`expect(wrapper.find('button.next')).to.have.prop('disabled', true);`
https://github.com/producthunt/chai-enzyme#propkey-val